docs/changelog/100808.yaml

```
@@ -0,0 +1,5 @@
+pr: 100808
+summary: Make `GetCheckpointAction` and `GetCheckpointNodeAction` cancellable
```

I don't think we should mention class names from the source code in the release notes.

```suggestion
summary: Make tasks that calculate checkpoints cancellable
```